src/main/java/com/shippingcostestimator/enterprise/dao/ShipmentSQLDAO.java

```diff
@@ -26,12 +26,12 @@ public Shipment saveEstimate(Shipment shipment) {
         return createdShipment;
     }
 
-    //In this case, .findAll(); doesn't return a List object, but instead an Iterable.
-    //To remedy this, we have two options.
-    //One, we can change all the methods involving fetchAllShipments to take an Iterable instead of a List.
-    //Or two, we do what I did below, where we converted it from an Iterable to a List.
-    //The professor suggests that for smaller projects, the second method is a good idea.
-    //He also said that for larger projects, it would be better to bite the bullet and do the first method.
+    /* In this case, .findAll(); doesn't return a List object, but instead an Iterable.
+     To remedy this, we have two options.
+      One, we can change all the methods involving fetchAllShipments to take an Iterable instead of a List.
+      Or two, we do what I did below, where we converted it from an Iterable to a List.
+      The professor suggests that for smaller projects, the second method is a good idea.
+      He also said that for larger projects, it would be better to bite the bullet and do the first method. */
```

Comment:
Changed the comment to a multi-line comment to make it more cleaner. [Source](https://www.baeldung.com/java-clean-code)

Reply:
If it truly is JavaDoc, use JavaDoc format:
https://www.tutorialspoint.com/java/java_documentation.htm
https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html